src/hidimstat/dcrt.py

```diff
@@ -1,307 +1,410 @@
+    implementation at: https://github.com/moleibobliu/Distillation-CRT/
 
     Parameters
     ----------
-    X : array-like of shape (n_samples, n_features)
-        Training data
-    y : array-like of shape (n_samples,)
-        Target values
-    estimated_coef : array-like of shape (n_features,), optional
-        Pre-computed feature coefficients
-    sigma_X : array-like of shape (n_features, n_features), optional
-        Covariance matrix of X
+    estimated_coef : array-like of shape (n_features,)
+        Pre-computed feature coefficients. default=None
+    sigma_X : array-like of shape (n_features, n_features) or None
+        Covariance matrix of X. default = None
     params_lasso_screening : dict
-        Parameters for main Lasso estimation or crossvalidation Lasso, including:
-        - alpha : float, optional - L1 regularization strength. If None, determined by CV.
-        - n_alphas : int, default=0 - Number of alphas for cross-validation.
+        - alpha : float - L1 regularization strength. If None, determined by CV.
+        - n_alphas : int, default=10 - Number of alphas for cross-validation.
         - alphas : array-like, default=None - List of alpha values to try in CV.
         - alpha_max_fraction : float, default=0.5 - Scale factor for alpha_max.
         For other parameters see :py:func:LassoCV, here is some advise configuration
         - cv : int, default=5 - Number of cross-validation folds.
-        - tol : float, default=1e-6 - Tolerance for optimization.
+        - tol : float, default=1e-6 - Convergence tolerance.
         - max_iter : int, default=1000 - Maximum iterations.
         - fit_intercept : bool, default=False - Whether to fit intercept.
         - selection : str, default='cyclic' - Feature selection method.
-    params_lasso_distillation_x : dict, optional
-        Parameters for X distillation Lasso. Defaults to params_lasso_screening.
-    params_lasso_distillation_y : dict, optional
-        Parameters for y distillation Lasso. Defaults to params_lasso_screening.
+    params_lasso_distillation_x : dict
+        Parameters for X distillation Lasso. If None, uses params_lasso_screening.
+    params_lasso_distillation_y : dict
+        Parameters for y distillation Lasso. If None, uses params_lasso_screening.
     refit : bool, default=False
-        Whether to refit on estimated support set
+        Whether to refit model on selected features
     screening : bool, default=True
-        Whether to screen variables
+        Whether to perform variable screening step. This selection is based on the coefficients of a cross-fitted Lasso model, keeping only the fraction (defined by screening_threshold) of variables with the largest coefficient and assigning a p-value of 1 to all other covariates.
     screening_threshold : float, default=0.1
-        Threshold for variable screening (0-100)
+        Percentile threshold for screening (0-100)
```

Comment:
If it's a percentile, it is surprising that the default is so low. There is something fishy here.

Reply:
You are right, it should be 10 and not 0.1

Reply:
```suggestion
        Percentile threshold for screening (0-100), larger values lead to the inclusion of more variables at the screening stage (screening_threshold=100 keeps all variables). 
```

Reply:
At the end, I removed the screening part for random_forest.
Lasso doesn't make the difference in the classification or regression problems.